package.json

```diff
@@ -35,6 +36,8 @@
     ]
   },
   "devDependencies": {
+    "eslint-config-airbnb": "^17.1.0",
```

Comment:
according to this we need to add the peerDeps too https://github.com/airbnb/javascript/tree/master/packages/eslint-config-airbnb#eslint-config-airbnb-1